### PR TITLE
Feature/logs

### PR DESCRIPTION
**Pull request by collindutter**

### CODE REVIEW

File: example-client/client.py [hunk 1]
```diff
@@ -29,26 +29,36 @@ def run_structure(input: str) -> str:
     structure_run = create_structure_run(GT_STRUCTURE_ID, {}, [input])
 
     # Runs are asynchronous, so we need to poll the status until it's no longer running.
-    run_id = structure_run["run_id"]
+    structure_run_id = structure_run["structure_run_id"]
     status = structure_run["status"]
-    printed_events = set()
-    while status == "RUNNING":
-        structure_run = get_structure_run(run_id)
+    printed_events = set()  # Keep track of which events we've printed.
```
Comment (SavagePencil) [inline example-client/client.py]: Just confirmed that Python `set` objects are _UNORDERED_. Will this result in out-of-order events? Why not a list?

**Review by SavagePencil — CHANGES_REQUESTED** [review]

File: example-client/client.py (hunk 1 above)
Comment (collindutter) [reply example-client/client.py]: Because we're just tracking event _ids_. Set over list for _speed_.

Comment (SavagePencil) [reply example-client/client.py]: can we call it `printed_event_ids` then 😤

Comment (collindutter) [reply example-client/client.py]: Done

**Review by SavagePencil — APPROVED** [review]
😤

**Pull request merged**